README.md

```diff
@@ -9,7 +9,7 @@
 [![Release](https://img.shields.io/github/v/release/mattermost/mattermost-plugin-aws-SNS)](https://github.com/mattermost/mattermost-plugin-aws-SNS/releases/latest)
 [![HW](https://img.shields.io/github/issues/mattermost/mattermost-plugin-aws-SNS/Up%20For%20Grabs?color=dark%20green&label=Help%20Wanted)](https://github.com/mattermost/mattermost-plugin-aws-SNS/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22Up+For+Grabs%22+label%3A%22Help+Wanted%22)
 
-**Maintainer:** [@mickmister](https://github.com/mickmister)
+**Maintainer:** [@wiggin77](https://github.com/wiggin77)
 **Co-Maintainer:** [@jfrerich](https://github.com/jfrerich)
```

Comment:
Let's remove that block. It indicates that the repo is maintained.

Reply:
Sure, done @hanzei